docs/sources/datasources/cloudmonitoring.md

```diff
@@ -226,6 +226,24 @@ The Alias By field allows you to control the format of the legend keys for SLO q
 
 SLO queries use the same [alignment period functionality as metric queries]({{< relref "#metric-queries" >}}).
 
+### MQL (Monitoring Query Language) queries
+
+> Only available in Grafana v7.x+
+
+The MQL query builder in the Google Cloud Monitoring data source allows you to display MQL result in time series format. To get an understanding of the basic concepts in MQL, please refer to Google Cloud Monitoring's [official docs](https://cloud.google.com/monitoring/mql).
+
+#### How to create an MQL query
+
+To create an MQL query, follow these steps:
+
+1. Choose the option **Monitoring Query Language (MQL)** in the **Query Type** dropdown.
```

Comment:
```suggestion
1. In the **Query Type** list, click **Monitoring Query Language (MQL)**.
```

Reply:
When writing instructions, it is a good idea to tell people where to do the thing before you tell them what to do.